src/main/java/org/opentripplanner/updater/alerts/GtfsRealtimeAlertsUpdater.java

```diff
@@ -90,7 +95,7 @@ protected void runPolling() {
                 throw new RuntimeException("Failed to get data from url " + url);
             }
 
-            final FeedMessage feed = FeedMessage.PARSER.parseFrom(data);
+            final FeedMessage feed = parseJson(IOUtils.toString(data));
```

Comment:
I don't think we want to remove the existing PB support. Instead, we should additionally support fully parsing the enhanced JSON feed via the `real-time-alerts-enhanced` updater.

Reply:
If they have the same data (plus few extra things in enhanced json) and we're doing full parsing of json anyway, then why would we need to keep pb support in place?

Reply:
I'd like to be able to submit these changes upstream at some point, and I doubt they'll accept a PR which removes support for PB files :)

Reply:
Got it, good point, fixed.